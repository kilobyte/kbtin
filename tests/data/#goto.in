#route a b {#showme A>B}
#route b c {#showme B>C}
#goto A B;#showme -=-
#goto a b;#showme -=-
#goto a c;#showme -=-
#route a c {#showme A>C} 5
#goto a c;#showme -=-
#route a c {#showme A>C} 21
#goto a c;#showme -=-
#goto a {};#showme -=-
#goto {} b;#showme -=-
#goto {} {};#showme -=-
a>b;#showme -=-
>c;#showme -=-
#showme ($loc)
