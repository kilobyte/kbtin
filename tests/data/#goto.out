#Location not found: [A]
-=-
A>B
-=-
A>B
B>C
-=-
A>C
-=-
A>B
B>C
-=-
#SYNTAX: #goto <from> <to>
-=-
#SYNTAX: #goto <from> <to>
-=-
#SYNTAX: #goto <from> <to>
-=-
A>B
-=-
B>C
-=-
(c)
